clarity.d.ts

```diff
@@ -58,6 +58,7 @@ interface IEnvelope {
   clarityId: string;
   impressionId: string;
   url: string;
+  title: string;
```

Comment:
You're right and let's hold off on this change until we fix the overall design to not send same data multiple times. Url, title, etc. all add up. Thanks for reviewing.